recipes/paho-mqtt-c/all/patches/0005-fix-cmake-install-for-1-3-8.patch

```diff
@@ -0,0 +1,544 @@
+diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
+index 4137d38..9f71d09 100644
+--- a/src/CMakeLists.txt
++++ b/src/CMakeLists.txt
+@@ -83,102 +83,162 @@ ADD_LIBRARY(common_obj OBJECT ${common_src})
+ SET_TARGET_PROPERTIES(common_obj PROPERTIES
+     POSITION_INDEPENDENT_CODE ON
+     COMPILE_DEFINITIONS "PAHO_MQTT_EXPORTS=1")
+-ADD_EXECUTABLE(MQTTVersion MQTTVersion.c)
+-SET_TARGET_PROPERTIES(MQTTVersion PROPERTIES
+-    POSITION_INDEPENDENT_CODE ON
+-    COMPILE_DEFINITIONS "PAHO_MQTT_IMPORTS=1")
+ ENDIF()
+ 
+-IF (PAHO_BUILD_STATIC)    
++IF (PAHO_BUILD_STATIC)
+ ADD_LIBRARY(common_obj_static OBJECT ${common_src})
+ SET_TARGET_PROPERTIES(common_obj_static PROPERTIES
+     POSITION_INDEPENDENT_CODE ON
```

Comment:
```suggestion
```

Hard to say without digging in depth but I think it should be removed to honor fPIC=False if static. It might need to be removed in other parts (basically, all libraries targets which don't link into a shared lib).